draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1147,26 +1148,29 @@ requirement above).
 
 If the log is unable to process a client's request, it MUST return an HTTP
 response code of 4xx/5xx (see [RFC7231]), and, in place of the responses
-outlined in the subsections below, the body SHOULD be a JSON structure
-containing at least the following field:
-
-error_message:
+outlined in the subsections below, the body SHOULD be a JSON Problem Details
+Object (see [RFC7807] Section 3), containing:
+
+type:
+: A URN reference identifying the problem. To facilitate automated diagnostics
+  of errors, this document defines a set of standard tokens for use in the
+  `type` field, within the URN namespace of: "urn:ietf:params:trans:error:".
+  Other than the generic codes detailed here, each error type is specific to
```

Comment:
We're only specifying one generic error: "malformed".

I suggest: s/the generic codes detailed here/the generic "malformed" error type defined in this section/

Reply:
Done.